spec/models/bridge_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Bridge, type: :model do
+  it 'saves' do
+    bridge = Bridge.create(name: 'My First Bridge', payload: '', inboundURL: 'https://bridgeapi.dev/b1234/inbound', outboundURL: 'https://wowservice.io/new/23847923864', method: 'post', retries: 5, delay: 15)
+    expect(Bridge.first).to eq bridge
+  end
+
+  it 'has many env vars' do 
+    bridge = Bridge.create(name: 'My First Bridge', payload: '', inboundURL: 'https://bridgeapi.dev/b1234/inbound', outboundURL: 'https://wowservice.io/new/23847923864', method: 'post', retries: 5, delay: 15)
+
+    bridge.env_vars << EnvVar.create(key: 'database', value: 'a102345ij2')
+    bridge.env_vars << EnvVar.create(key: 'database_password', value: 'supersecretpasswordwow')
+
+    expect(bridge.env_vars.count).to eq 2
```

Comment:
This is what I am talking about. Can you do `expect(Bridge.find(bridge.id)).to eq 2`? I have a feeling that would fail meaning `<<` is only the `Array#<<`. Try it out. If it succeeds, that's awesome.